dashboard/app/views/lessons/show.html.haml

```diff
@@ -0,0 +1,10 @@
+%h1= @lesson.name
```

Comment:
Are you going to set this up into react too or is that follow up work we should plan to tackle later?

Reply:
this should get converted into react. I would be happy to tackle that too in the next PR.